Cron_Philip/Assignments/controllingUsers/controlUsers.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html ng-app="app">
+<head>
+    <meta charset="utf-8">
+    <title>Controlling Users</title>
+    <!-- bootstrap -->
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+
+    <!-- angular -->
+
+    <script src = 'bower_components/angular/angular.js'></script>
+    <script>
+
+        var app = angular.module('app', []);
+        app.controller('usersController', function() {
+
+            this.users = [];
+
+            this.createUser = function() {
+                this.users.push(this.newUser);
```

Comment:
this would be a good place to first check to ensure this.newUser exist. What happens if the user clicks 'addUser' button w/o filling out anything in the input text boxes? angular won't have created the this.newUser object yt, then boom! headshot. there goes ya server, son!